src/styles/content.scss

```diff
@@ -275,4 +275,9 @@ table.os-raise-horizontaltable {
 
 .os-raise-text-bold {
   font-weight: 700;
+}
+
+.os-raise-img-fluid {
```

Comment:
We could potentially use it for `<video>` as well, so that's one reason I kept it "open" (maybe something like `os.-raise-media-{somethingCleverHere}`. You also increase the specificity I believe by adding the `img` prefix. If we think it might get used outside of its use-case then I am open to narrowing it down to your suggestion.

Reply:
That makes sense. I'm fine with leaving open (either with the current name or a different one based on your preference).